examples/Example.h

```diff
@@ -229,10 +234,13 @@ struct Window
             if (needResize) {
                 resize();
                 needResize = false;
+                if (example->update(canvas, elapsed)) {
+                    needDraw = true;
+                }
             }
 
-            if (tickCnt > 0) {
-                if (example->update(canvas, elapsed)) {
+            if (example->isAnimation && tickCnt > 0) {
+                if (!needDraw && example->update(canvas, elapsed)) {
```

Comment:
I think needDraw itself works for resizing as well, isAnimation flag doesn't need.

Reply:
The example that does not need update per frame did not need to call example->update(,) in every loop, and I added isAnimation to distinguish it. I thought this was a way to make the code less complicated.

My intention(?) is... for check detailed small changes, In some examples needs when the window size is resized the result resized to fit the window size.

Reply:
> Only the flag can be added here.

If canvas->update() is not called in no animation example, a crash will occur when the window size becomes smaller than the canvas.

Reply:
> The example that does not need update per frame

`needDraw` doing this.

Reply:
> > Only the flag can be added here.
> 
> If canvas->update() is not called in no animation example, a crash will occur when the window size becomes smaller than the canvas.

It might be related to an SW engine problem that hasn't been addressed before. I have already observed it and decided to address it correctly later.

Reply:
@JSUYA Sorry, i might need to review this again, probably, i confused with 'tickCnt' by my dead brain.

Reply:
> If canvas->update() is not called in no animation example,

@JSUYA just reviewed code again, `canvas->update()` is always called, so what is your problem?

Reply:
1. If the log option is turned on, unnecessary logs are generated.
2. Of course, redraw does not occur because RenderUpdateFlag does not change internally, but I don't think there is a need to travel all paints's update(). I thought it was an unnecessary action for examples that did not implement `Example::update( , )`.

Reply:
> 1. If the log option is turned on, unnecessary logs are generated.

Sorry i don't understand the context why suddenly "log" come from in this PR.


> 2\. Of course, redraw does not occur because RenderUpdateFlag does not change internally, but I don't think there is a need to travel all paints's update(). I thought it was an unnecessary action for examples that did not implement `Example::update( , )`.

 Example::update() is an interface to be triggered every frame. Each examples could implement it when they want to update the scene by their demands.

Reply:
I updated PR!!
The real problem is that I tried to call canvas->update() in Example::update(, ). Because of this, I had to distinguish between animated examples.

+) Regarding the mention of log,,,, for the above reason, when running the no animated example, the log of update() was called every loop. But it's okay now.